lib/membrane_rtmp_plugin/rtmp/sink/sink.ex

```diff
@@ -12,16 +12,17 @@ defmodule Membrane.RTMP.Sink do
   require Membrane.Logger
 
   alias __MODULE__.Native
-  alias Membrane.{AAC, MP4}
+  alias Membrane.{AAC, MP4, Buffer}
 
   @supported_protocols ["rtmp://", "rtmps://"]
   @connection_attempt_interval 500
   @default_state %{
     attempts: 0,
     native: nil,
-    buffered_frame: nil,
+    # Keys here are the pad names.
+    frame_buffer: %{audio: nil, video: nil},
     ready: false,
```

Comment:
I know you didn't change this code, but you can change this by the way.
```suggestion
    ready?: false,
```